private/doc.go

```diff
@@ -0,0 +1,3 @@
+// Package private contains helpers private to github.com/containers/image, with no intent or commitment to make them useful for any other purpose.
+// Do not use outside!
+package private
```

Comment:
> and/or `docker`

Ooh, yes; we can treat this as “layered transport“ (OCI, OpenShift) reusing functionality from an underlying one (dir, Docker). That’s actually pretty clean, good idea.